src/ButtonWithBadge/ButtonWithBadge.tsx

```diff
@@ -8,12 +9,12 @@ export interface ButtonWithBadgeProps {
 }
 
 export const ButtonWithBadge: React.FC<ButtonWithBadgeProps> = ({ children, badge, onClick }) => (
-  <Fragment>
-    <StyledButton onClick={onClick} variant="white" size="standard" radius="round">
+  <StyledContainer>
+    <ButtonV2 onClick={onClick} variant="white" size="standard" radius="round">
       {children}
-    </StyledButton>
+    </ButtonV2>
     <StyledNewBadge color="#ffffff" background="#2818f9" size="small">
       {badge}
     </StyledNewBadge>
```

Comment:
If you put this inside the button, add `position: relative;` to `StyledButton` and then you can delete the `StyledContainer` - then you should be good to go.

Reply:
Thank you 🙏  hope it's ok now